payments-ui-core/src/main/java/com/stripe/android/ui/core/forms/resources/LpmRepository.kt

```diff
@@ -103,7 +103,10 @@ class LpmRepository constructor(
         serverLpmSpecs: String?,
         force: Boolean = false
     ) {
-        if (!isLoaded() || force) {
+        // If the expectedLpms is different form last time, we still need to reload.
+        var lpmsNotParsedFromServerSpec = expectedLpms
```

Comment:
If we get passed two different PMs with different values, we might need to reparse in the library to deserialize the new LPMs that we need.